common/lib/capa/capa/tests/test_files/extended_hints_checkbox.xml

```diff
@@ -0,0 +1,108 @@
+<problem schema="edXML/1.0">
```

Comment:
so - here you are defining a schema. Is that a part of your work or hold over from the last project?